common/ledger.js

```diff
@@ -1,18 +1,34 @@
 import { getHDPath } from './hdpath'
 import network from './network'
 
-export async function getLedger() {
+export async function getLedger(isWindows, hasHIDEnabled) {
```

Comment:
why pass this in here? you can calculate both variables in here right?

Reply:
well, that is true